ReflectometryServer/parameters.py

```diff
@@ -438,7 +438,7 @@ def __init__(self, name, angled_component, sim=False, init=0, description=None,
         if self._set_point_rbv is None:
             self._reflection_component.beam_path_set_point.add_listener(InitUpdate, self._initialise_sp_from_motor)
 
-        self._reflection_component.beam_path_rbv.add_listener(BeamPathUpdate, self._on_update_rbv)
+        self._reflection_component.beam_path_rbv.add_listener(PhysicalMoveUpdate, self._on_update_rbv)
```

Comment:
We should talk about this.
1. I think that reflecting component should raise beam path calc change on angle change like displacement. Beampath update should be called when the beam changes through the component
2. We should have a test which catches this.

Reply:
In retropsect this need both triggers.